Index: optware/make/py-psycopg2.mk
===================================================================
--- optware/make/py-psycopg2.mk	(revision 12863)
+++ optware/make/py-psycopg2.mk	(working copy)
@@ -21,7 +21,7 @@
 # from your name or email address.  If you leave MAINTAINER set to
 # "NSLU2 Linux" other developers will feel free to edit.
 #
-PY-PSYCOPG2_SITE=http://initd.org/pub/software/psycopg
+PY-PSYCOPG2_SITE=http://initd.org/psycopg/tarballs/PSYCOPG-2-3
 PY-PSYCOPG2_VERSION=2.3.2
 PY-PSYCOPG2_SOURCE=psycopg2-$(PY-PSYCOPG2_VERSION).tar.gz
 PY-PSYCOPG2_DIR=psycopg2-$(PY-PSYCOPG2_VERSION)
Index: optware/make/py-nevow.mk
===================================================================
--- optware/make/py-nevow.mk	(revision 12863)
+++ optware/make/py-nevow.mk	(working copy)
@@ -84,7 +83,6 @@
 # then it will be fetched from the site using wget.
 #
 $(DL_DIR)/$(PY-NEVOW_SOURCE):
-	$(WGET) -O $@ $(PY-NEVOW_SITE) || \
 	$(WGET) -P $(@D) $(SOURCES_NLO_SITE)/$(@F)
 
 #
Index: optware/make/util-linux.mk
===================================================================
--- optware/make/util-linux.mk	(revision 12863)
+++ optware/make/util-linux.mk	(working copy)
@@ -20,7 +20,7 @@
 # from your name or email address.  If you leave MAINTAINER set to
 # "NSLU2 Linux" other developers will feel free to edit.
 #
-UTIL_LINUX_SITE=ftp://ftp.kernel.org/pub/linux/utils/util-linux
+UTIL_LINUX_SITE=ftp://ftp.kernel.org/pub/linux/utils/util-linux/v2.12
 UTIL_LINUX_VERSION=2.12r
 UTIL_LINUX_SOURCE=util-linux-$(UTIL_LINUX_VERSION).tar.gz
 UTIL_LINUX_DIR=util-linux-$(UTIL_LINUX_VERSION)
Index: optware/make/rtpproxy.mk
===================================================================
--- optware/make/rtpproxy.mk	(revision 12863)
+++ optware/make/rtpproxy.mk	(working copy)
@@ -79,7 +79,7 @@
 # then it will be fetched from the site using wget.
 #
 $(DL_DIR)/$(RTPPROXY_SOURCE):
-	$(WGET) -P $(@D) $(RTPROXY_SITE)/$(@F) || \
+	$(WGET) -P $(@D) $(RTPPROXY_SITE)/$(@F) || \
 	$(WGET) -P $(@D) $(SOURCES_NLO_SITE)/$(@F)
 
 #
Index: optware/make/atop.mk
===================================================================
--- optware/make/atop.mk	(revision 12863)
+++ optware/make/atop.mk	(working copy)
@@ -10,7 +10,7 @@
 # questions. But feel free to update or change this package
 # if there are reasons.
 #
-ATOP_SITE=http://www.atconsultancy.nl/atop/packages
+ATOP_SITE=http://www.atoptool.nl/download
 ATOP_VERSION=1.21
 ATOP_SOURCE=atop-$(ATOP_VERSION).tar.gz
 ATOP_DIR=atop-$(ATOP_VERSION)
Index: optware/make/py-pygresql.mk
===================================================================
--- optware/make/py-pygresql.mk	(revision 12863)
+++ optware/make/py-pygresql.mk	(working copy)
@@ -21,7 +21,7 @@
 # from your name or email address.  If you leave MAINTAINER set to
 # "NSLU2 Linux" other developers will feel free to edit.
 #
-PY-PYGRESQL_SITE=ftp://ftp.PyGreSQL.org/pub/distrib
+PY-PYGRESQL_SITE=http://PyGreSQL.org/files
 PY-PYGRESQL_VERSION=3.8.1
 PY-PYGRESQL_SOURCE=PyGreSQL-$(PY-PYGRESQL_VERSION).tgz
 PY-PYGRESQL_DIR=PyGreSQL-$(PY-PYGRESQL_VERSION)
Index: optware/make/py-turbogears.mk
===================================================================
--- optware/make/py-turbogears.mk	(revision 12863)
+++ optware/make/py-turbogears.mk	(working copy)
@@ -21,7 +21,7 @@
 # from your name or email address.  If you leave MAINTAINER set to
 # "NSLU2 Linux" other developers will feel free to edit.
 #
-PY-TURBOGEARS_SITE=http://files.turbogears.org/eggs
+PY-TURBOGEARS_SITE=http://pypi.python.org/packages/source/T/TurboGears
 PY-TURBOGEARS_VERSION=1.1
 PY-TURBOGEARS_IPK_VERSION=1
 PY-TURBOGEARS_SOURCE=TurboGears-$(PY-TURBOGEARS_VERSION).tar.gz
Index: optware/make/libcapi20.mk
===================================================================
--- optware/make/libcapi20.mk	(revision 12863)
+++ optware/make/libcapi20.mk	(working copy)
@@ -19,7 +19,7 @@
 #
 # You should change all these variables to suit your package.
 #
-LIBCAPI20_SITE=ftp://ftp.melware.net/capi-utils
+LIBCAPI20_SITE=http://www.openadk.org/distfiles
 LIBCAPI20_VERSION=3.0.5-cm
 LIBCAPI20_SOURCE=libcapi20-$(LIBCAPI20_VERSION).tar.gz
 LIBCAPI20_DIR=libcapi20-$(LIBCAPI20_VERSION)
Index: optware/make/dash.mk
===================================================================
--- optware/make/dash.mk	(revision 12863)
+++ optware/make/dash.mk	(working copy)
@@ -20,7 +20,7 @@
 # from your name or email address.  If you leave MAINTAINER set to
 # "NSLU2 Linux" other developers will feel free to edit.
 #
-DASH_SITE=http://ftp.debian.org/debian/pool/main/d/dash
+DASH_SITE=https://launchpad.net/debian/+archive/primary/+files
 DASH_VERSION=0.5.6.1
 DASH_SOURCE=dash_$(DASH_VERSION).orig.tar.gz
 DASH_DIR=dash-$(DASH_VERSION)
Index: optware/make/autossh.mk
===================================================================
--- optware/make/autossh.mk	(revision 12863)
+++ optware/make/autossh.mk	(working copy)
@@ -20,7 +20,7 @@
 # from your name or email address.  If you leave MAINTAINER set to
 # "NSLU2 Linux" other developers will feel free to edit.
 #
-AUTOSSH_SITE=http://www.harding.motd.ca/autossh
+AUTOSSH_SITE=http://www.sfr-fresh.com/unix/privat
 AUTOSSH_VERSION=1.4b
 AUTOSSH_SOURCE=autossh-$(AUTOSSH_VERSION).tgz
 AUTOSSH_DIR=autossh-$(AUTOSSH_VERSION)
Index: optware/make/postgresql.mk
===================================================================
--- optware/make/postgresql.mk	(revision 12863)
+++ optware/make/postgresql.mk	(working copy)
@@ -27,7 +27,7 @@
 # "NSLU2 Linux" other developers will feel free to edit.
 #
 POSTGRESQL_VERSION=8.2.13
-POSTGRESQL_SITE=ftp://ftp.postgresql.org/pub/source/v$(POSTGRESQL_VERSION)
+POSTGRESQL_SITE=http://ftp.riken.go.jp/pub/FreeBSD/distfiles/postgresql
 POSTGRESQL_SOURCE=postgresql-$(POSTGRESQL_VERSION).tar.bz2
 POSTGRESQL_DIR=postgresql-$(POSTGRESQL_VERSION)
 POSTGRESQL_UNZIP=bzcat
Index: optware/make/py-mantissa.mk
===================================================================
--- optware/make/py-mantissa.mk	(revision 12863)
+++ optware/make/py-mantissa.mk	(working copy)
@@ -81,7 +80,6 @@
 # then it will be fetched from the site using wget.
 #
 $(DL_DIR)/$(PY-MANTISSA_SOURCE):
-	$(WGET) -O $@ $(PY-MANTISSA_SITE) || \
 	$(WGET) -P $(@D) $(SOURCES_NLO_SITE)/$(@F)
 
 #
Index: optware/make/picolisp.mk
===================================================================
--- optware/make/picolisp.mk	(revision 12863)
+++ optware/make/picolisp.mk	(working copy)
@@ -76,7 +75,6 @@
 # then it will be fetched from the site using wget.
 #
 $(DL_DIR)/$(PICOLISP_SOURCE):
-	$(WGET) -O $(DL_DIR)/$(PICOLISP_SOURCE) "$(PICOLISP_SITE)$(PICOLISP_SOURCE)" || \
 	$(WGET) -P $(DL_DIR) $(SOURCES_NLO_SITE)/$(PICOLISP_SOURCE)
 
 #
Index: optware/make/mdadm.mk
===================================================================
--- optware/make/mdadm.mk	(revision 12863)
+++ optware/make/mdadm.mk	(working copy)
@@ -19,7 +19,7 @@
 #
 # You should change all these variables to suit your package.
 #
-MDADM_SITE=http://www.cse.unsw.edu.au/~neilb/source/mdadm/
+MDADM_SITE=http://www.kernel.org/pub/linux/utils/raid/mdadm/
 MDADM_VERSION=2.6
 MDADM_SOURCE=mdadm-$(MDADM_VERSION).tgz
 MDADM_DIR=mdadm-$(MDADM_VERSION)
Index: optware/make/py-epsilon.mk
===================================================================
--- optware/make/py-epsilon.mk	(revision 12863)
+++ optware/make/py-epsilon.mk	(working copy)
@@ -81,7 +80,6 @@
 # then it will be fetched from the site using wget.
 #
 $(DL_DIR)/$(PY-EPSILON_SOURCE):
-	$(WGET) -O $@ $(PY-EPSILON_SITE) || \
 	$(WGET) -P $(@D) $(SOURCES_NLO_SITE)/$(@F)
 
 #
Index: optware/make/py-axiom.mk
===================================================================
--- optware/make/py-axiom.mk	(revision 12863)
+++ optware/make/py-axiom.mk	(working copy)
@@ -81,7 +80,6 @@
 # then it will be fetched from the site using wget.
 #
 $(DL_DIR)/$(PY-AXIOM_SOURCE):
-	$(WGET) -O $@ $(PY-AXIOM_SITE) || \
 	$(WGET) -P $(@D) $(SOURCES_NLO_SITE)/$(@F)
 
 #
Index: optware/make/quagga.mk
===================================================================
--- optware/make/quagga.mk	(revision 12863)
+++ optware/make/quagga.mk	(working copy)
@@ -26,7 +26,6 @@
 # from your name or email address.  If you leave MAINTAINER set to
 # "NSLU2 Linux" other developers will feel free to edit.
 #
-QUAGGA_SITE=http://www.quagga.net/download
 QUAGGA_VERSION=0.99.16
 QUAGGA_SOURCE=quagga-$(QUAGGA_VERSION).tar.gz
 QUAGGA_DIR=quagga-$(QUAGGA_VERSION)
@@ -82,7 +81,6 @@
 # then it will be fetched from the site using wget.
 #
 $(DL_DIR)/$(QUAGGA_SOURCE):
-	$(WGET) -P $(@D) $(QUAGGA_SITE)/$(@F) || \
 	$(WGET) -P $(@D) $(SOURCES_NLO_SITE)/$(@F)
 
 #
Index: optware/make/knock.mk
===================================================================
--- optware/make/knock.mk	(revision 12863)
+++ optware/make/knock.mk	(working copy)
@@ -4,7 +4,6 @@
 # 
 ###########################################################
 #
-KNOCK_SITE=http://www.zeroflux.org/knock/files
 KNOCK_VERSION=0.5
 KNOCK_SOURCE=knock-$(KNOCK_VERSION).tar.gz
 KNOCK_DIR=knock-$(KNOCK_VERSION)
@@ -53,7 +52,6 @@
 # then it will be fetched from the site using wget.
 #
 $(DL_DIR)/$(KNOCK_SOURCE):
-	$(WGET) -P $(@D) $(KNOCK_SITE)/$(@F) || \
 	$(WGET) -P $(@D) $(SOURCES_NLO_SITE)/$(@F)
 
 #
Index: optware/make/dircproxy.mk
===================================================================
--- optware/make/dircproxy.mk	(revision 12863)
+++ optware/make/dircproxy.mk	(working copy)
@@ -20,7 +20,6 @@
 # from your name or email address.  If you leave MAINTAINER set to
 # "NSLU2 Linux" other developers will feel free to edit.
 #
-DIRCPROXY_SITE=http://dircproxy.securiweb.net/pub/1.1
 DIRCPROXY_VERSION=1.1.0
 DIRCPROXY_SOURCE=dircproxy-$(DIRCPROXY_VERSION).tar.gz
 DIRCPROXY_DIR=dircproxy-$(DIRCPROXY_VERSION)
@@ -76,7 +75,6 @@
 # then it will be fetched from the site using wget.
 #
 $(DL_DIR)/$(DIRCPROXY_SOURCE):
-	$(WGET) -P $(DL_DIR) $(DIRCPROXY_SITE)/$(@F) || \
 	$(WGET) -P $(DL_DIR) $(SOURCES_NLO_SITE)/$(@F)
 
 #
Index: optware/make/libhid.mk
===================================================================
--- optware/make/libhid.mk	(revision 12863)
+++ optware/make/libhid.mk	(working copy)
@@ -20,7 +20,6 @@
 # from your name or email address.  If you leave MAINTAINER set to
 # "NSLU2 Linux" other developers will feel free to edit.
 #
-LIBHID_SITE=https://alioth.debian.org/frs/download.php/1958
 LIBHID_VERSION=0.2.16
 LIBHID_SOURCE=libhid-$(LIBHID_VERSION).tar.gz
 LIBHID_DIR=libhid-$(LIBHID_VERSION)
@@ -80,7 +79,6 @@
 # then it will be fetched from the site using wget.
 #
 $(DL_DIR)/$(LIBHID_SOURCE):
-	$(WGET) --no-check-certificate -P $(@D) $(LIBHID_SITE)/$(@F) || \
 	$(WGET) -P $(@D) $(SOURCES_NLO_SITE)/$(@F)
 
 #
Index: optware/make/calc.mk
===================================================================
--- optware/make/calc.mk	(revision 12863)
+++ optware/make/calc.mk	(working copy)
@@ -20,7 +20,6 @@
 # from your name or email address.  If you leave MAINTAINER set to
 # "NSLU2 Linux" other developers will feel free to edit.
 #
-CALC_SITE=http://www.isthe.com/chongo/src/calc
 CALC_VERSION=2.12.4.0
 CALC_SOURCE=calc-$(CALC_VERSION).tar.bz2
 CALC_DIR=calc-$(CALC_VERSION)
@@ -76,7 +75,6 @@
 # then it will be fetched from the site using wget.
 #
 $(DL_DIR)/$(CALC_SOURCE):
-	$(WGET) -P $(@D) $(CALC_SITE)/$(@F) || \
 	$(WGET) -P $(@D) $(SOURCES_NLO_SITE)/$(@F)
 
 #
Index: optware/make/minicom.mk
===================================================================
--- optware/make/minicom.mk	(revision 12863)
+++ optware/make/minicom.mk	(working copy)
@@ -26,7 +26,6 @@
 # from your name or email address.  If you leave MAINTAINER set to
 # "NSLU2 Linux" other developers will feel free to edit.
 #
-MINICOM_SITE=http://alioth.debian.org/frs/download.php/3487
 MINICOM_VERSION=2.5
 MINICOM_SOURCE=minicom-$(MINICOM_VERSION).tar.gz
 MINICOM_DIR=minicom-$(MINICOM_VERSION)
@@ -86,7 +85,6 @@
 # then it will be fetched from the site using wget.
 #
 $(DL_DIR)/$(MINICOM_SOURCE):
-	$(WGET) -P $(@D) $(MINICOM_SITE)/$(@F) || \
 	$(WGET) -P $(@D) $(SOURCES_NLO_SITE)/$(@F)
 
 #
Index: optware/make/eventlog.mk
===================================================================
--- optware/make/eventlog.mk	(revision 12863)
+++ optware/make/eventlog.mk	(working copy)
@@ -4,7 +4,6 @@
 #
 ###########################################################
 
-EVENTLOG_SITE=http://www.balabit.com/downloads/files/syslog-ng/sources/stable/src
 EVENTLOG_VERSION=0.2.7
 EVENTLOG_SOURCE=eventlog-$(EVENTLOG_VERSION).tar.gz
 EVENTLOG_DIR=eventlog-$(EVENTLOG_VERSION)
@@ -60,7 +59,6 @@
 # then it will be fetched from the site using wget.
 #
 $(DL_DIR)/$(EVENTLOG_SOURCE):
-	$(WGET) -P $(@D) $(EVENTLOG_SITE)/$(@F) || \
 	$(WGET) -P $(@D) $(SOURCES_NLO_SITE)/$(@F)
 
 #
Index: optware/make/stunnel.mk
===================================================================
--- optware/make/stunnel.mk	(revision 12863)
+++ optware/make/stunnel.mk	(working copy)
@@ -19,7 +19,6 @@
 #
 # You should change all these variables to suit your package.
 #
-STUNNEL_SITE=http://www.stunnel.org/download/stunnel/src
 STUNNEL_VERSION=4.26
 STUNNEL_SOURCE=stunnel-$(STUNNEL_VERSION).tar.gz
 STUNNEL_DIR=stunnel-$(STUNNEL_VERSION)
@@ -78,7 +77,6 @@
 # then it will be fetched from the site using wget.
 #
 $(DL_DIR)/$(STUNNEL_SOURCE):
-	$(WGET) -P $(@D) $(STUNNEL_SITE)/$(@F) || \
 	$(WGET) -P $(@D) $(SOURCES_NLO_SITE)/$(@F)
 
 #
Index: optware/make/py-gdchart2.mk
===================================================================
--- optware/make/py-gdchart2.mk	(revision 12863)
+++ optware/make/py-gdchart2.mk	(working copy)
@@ -21,7 +21,6 @@
 # from your name or email address.  If you leave MAINTAINER set to
 # "NSLU2 Linux" other developers will feel free to edit.
 #
-PY-GDCHART2_SITE=http://www.nullcube.com/software/pygdchart2
 PY-GDCHART2_VERSION=Beta1
 PY-GDCHART2_SOURCE=pygdchart2.tar.gz
 PY-GDCHART2_DIR=pygdchart2
@@ -81,7 +80,6 @@
 # then it will be fetched from the site using wget.
 #
 $(DL_DIR)/$(PY-GDCHART2_SOURCE):
-	$(WGET) -P $(DL_DIR) $(PY-GDCHART2_SITE)/$(@F) || \
 	$(WGET) -P $(DL_DIR) $(SOURCES_NLO_SITE)/$(@F)
 
 #
Index: optware/make/libijs.mk
===================================================================
--- optware/make/libijs.mk	(revision 12863)
+++ optware/make/libijs.mk	(working copy)
@@ -20,7 +20,6 @@
 # from your name or email address.  If you leave MAINTAINER set to
 # "NSLU2 Linux" other developers will feel free to edit.
 #
-LIBIJS_SITE=http://www.linuxprinting.org/ijs/download
 LIBIJS_VERSION=0.35
 LIBIJS_SOURCE=ijs-$(LIBIJS_VERSION).tar.bz2
 LIBIJS_DIR=ijs-$(LIBIJS_VERSION)
@@ -76,7 +75,6 @@
 # then it will be fetched from the site using wget.
 #
 $(DL_DIR)/$(LIBIJS_SOURCE):
-	$(WGET) -P $(DL_DIR) $(LIBIJS_SITE)/$(LIBIJS_SOURCE) || \
 	$(WGET) -P $(DL_DIR) $(SOURCES_NLO_SITE)/$(LIBIJS_SOURCE)
 
 #
Index: optware/Makefile
===================================================================
--- optware/Makefile	(revision 12932)
+++ optware/Makefile	(working copy)
@@ -552,7 +552,7 @@
 # The hostname or IP number of our local dl.sf.net mirror
 SOURCEFORGE_MIRROR=downloads.sourceforge.net
 #SOURCES_NLO_SITE=http://sources.nslu2-linux.org/sources
-SOURCES_NLO_SITE=http://ftp.osuosl.org/pub/nslu2/sources
+SOURCES_NLO_SITE=http://ipkg.nslu2-linux.org/sources
 
 TARGET_CXX=$(TARGET_CROSS)g++
 TARGET_CC=$(TARGET_CROSS)gcc
